src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/jpa/IncidentReport.kt

```diff
@@ -40,13 +41,17 @@ class IncidentReport(
   @Enumerated(EnumType.STRING)
   val incidentType: IncidentType,
 
+  var summary: String?,
   var incidentDetails: String,
 
   val reportedBy: String,
   val reportedDate: LocalDateTime,
   @Enumerated(EnumType.STRING)
   var status: IncidentStatus = IncidentStatus.DRAFT,
 
+  @ManyToOne(fetch = FetchType.LAZY, cascade = [CascadeType.ALL])
+  val event: IncidentEvent? = null,
```

Comment:
what do you think about making this non-nullable? it'd make the data model simpler i think. we could just have the typical case being 1 event ↔︎ 1 incident

Reply:
the statsy reports / consumers of our data wouldn't have to have branching for orphaned incidents vs those grouped into events

Reply:
Yes could do that - its just I need to create one for all NOMIS migrated records